htdocs/langs/fr_FR/bills.lang

```diff
@@ -384,6 +384,7 @@ WarningInvoiceDateInFuture=Attention, la date de facturation est antérieur à l
 WarningInvoiceDateTooFarInFuture=Attention, la date de facturation est trop éloignée de la date actuelle
 ViewAvailableGlobalDiscounts=Voir les remises disponibles
 GroupPaymentsByModOnReports=Paiements groupés par mode sur les rapports
+Payed=Règlement reçu
```

Comment:
PR on files into langs/xx_XX where xx_XX is not en_US is useless because files are erased by Transifex translation of en_US. Change on translation files must be done with PR for en_US only. For other languages (en_NZ, es_ES, ...), fix of translation must be done directly on https://www.transifex.com/dolibarr-association/dolibarr/